src/hooks/usePets.js

```diff
@@ -1,22 +1,31 @@
 import { useEffect, useState } from "react";
 import { GetPets } from "../services/ApiService";
 
-const usePets = initialEndPoint => {
+const usePets = endPoint => {
   const [isLoading, setIsLoading] = useState(true);
   const [hasError, setHasError] = useState(false);
-
+  const [petItems, setPetItems] = useState([]);
   useEffect(() => {
-    GetPets()
-      .then(({ records }) => {})
+    GetPets(endPoint)
+      .then(response => {
+        const { records } = response;
```

Comment:
after scrolling down to 

`const [{ petItems, isLoading }] = usePets(`/hub/pets/${animalId}`);`

i wonder if we should just return the response here, and then grab what we need from that inside of the component.

Reply:
@martypowell yeah either way is fine. It would remove adding logic in the hook to determine if its an array or a single object which is more consistent with how we've done others so Im okay with that.

Reply:
done